pkg/alerting/shared/streams.go

```diff
@@ -9,21 +9,65 @@ import (
 
```

Comment:
Any cortex/alertmanager-specific code should be confined to the plugins as much as possible; if you can, keep only the string constants here.

Reply:
Done